### ISSUE TITLE
Suggestion: Abort deleting a non-empty directory instead of crashing

### ISSUE THREAD

**Issue by Madrigalist** [RimSort/RimSort#438]
**Is your feature request related to a problem? Please describe.**
When a mod's subfolder is open in file explorer, RimSort will simply crash if it tries to delete the entire mod. 

**Describe the solution you'd like**
RimSort should show an error message and abort the operation.

**Comment by Madrigalist** [comment]
Possible fix by [437](https://github.com/RimSort/RimSort/pull/437)

**Comment by Raizanad** [comment]
Just to expand a bit on this, this would also happen if anything is using the file in question - from file explorer, to VS code, to RimWorld itself.

**Comment by hk21702** [comment]
Should have been fixed with #437.
